### PR TITLE
Timewarrior module

### PR DESCRIPTION
**Pull request by JindrichPilar**
Module for heads up information for taskwarrior.org/docs/timewarrior/

### CODE REVIEW

File: i3pystatus/timewarrior.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    )
+
+    def loadTrack(self):
+        try:
+            tracks_json = subprocess.check_output(['timew', 'export'])
+            tracks = loads(tracks_json.decode("utf-8"))
+            self.track = tracks[-1]
+
+        except ValueError:
+            print('Decoding JSON has failed')
```
Comment (facetoe) [inline i3pystatus/timewarrior.py]: Should rethrow an exception here, that way the error will be visible in the status bar and not break things.

Comment (JindrichPilar) [inline i3pystatus/timewarrior.py]: Error is now re-raised.

File: i3pystatus/timewarrior.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    )
+
+    def loadTrack(self):
+        try:
+            tracks_json = subprocess.check_output(['timew', 'export'])
+            tracks = loads(tracks_json.decode("utf-8"))
+            self.track = tracks[-1]
+
+        except ValueError as error:
+            print('Decoding JSON has failed')
```
Comment (enkore) [inline i3pystatus/timewarrior.py]: log instead of print

Comment (JindrichPilar) [inline i3pystatus/timewarrior.py]: Print changed to self.logger.exception.

**Comment by enkore** [comment]
Try adding dateutil.parser and dateutil.relativedata to docs/conf.py

See http://i3pystatus.readthedocs.io/en/latest/module.html#handling-dependencies

**Comment by JindrichPilar** [comment]
I have updated ``docs/conf.py``.

**Pull request merged**